src/uname/uname.rs

```diff
@@ -32,48 +47,76 @@ static HOST_OS: &'static str = "Darwin";
 static HOST_OS: &'static str = "Fuchsia";
 
 pub fn uumain(args: Vec<String>) -> i32 {
-    let mut opts = new_coreopts!(SYNTAX, SUMMARY, "");
 
-    opts.optflag("a",
-                 "all",
-                 "Behave as though all of the options -mnrsv were specified.");
-    opts.optflag("s", "sysname", "print the operating system name.");
-    opts.optflag("n", "nodename", "print the nodename (the nodename may be a name that the system is known by to a communications network).");
-    opts.optflag("r", "kernel-release", "print the operating system release.");
-    opts.optflag("v", "kernel-version", "print the operating system version.");
-    opts.optflag("m", "machine", "print the machine hardware name.");
+    let usage = format!("{} [OPTION]...", executable!());
+    let matches = App::new(executable!())
+        .version(VERSION)
+        .about(ABOUT)
+        .usage(&usage[..])
+        .arg(Arg::with_name(OPT_ALL)
+            .short("a")
+            .long(OPT_ALL)
+            .help("Behave as though all of the options -mnrsv were specified."))
+        .arg(Arg::with_name(OPT_KERNELNAME)
+            .short("s")
+            .long(OPT_KERNELNAME)
+            .alias("sysname") // Obsolescent option in GNU uname
+            .help("print the operating system name.")
```

Comment:
I think you deleted a `)` by accident when you rebased.

Reply:
Eep! Sorry about that.